src/igs_service.c

```diff
@@ -717,6 +717,11 @@ igs_result_t igsagent_service_call (igsagent_t *agent,
                              agent->definition->name, agent->uuid,
                              remote_agent->definition->name, service_name,
                              remote_agent->uuid);
+                zyre_shouts (agent->context->node, agent->igs_channel,
```

Comment:
See the other comment on duplicate messages.   
[src/igs_service.c L822](https://github.com/zeromq/ingescape/pull/6/commits/f3c01e5dfa25664922f4a1c0dab26fe6ab78c9fa#diff-d468bd1da851bfab82c47a5b806a08cfce8ac95b32e121be242ebaee44b247c8R822)